replicator/src/core.rs

```diff
@@ -29,8 +30,14 @@ pub async fn start_replicator() -> anyhow::Result<()> {
     // We set up the certificates and SSL mode.
     let mut trusted_root_certs = vec![];
     let ssl_mode = if replicator_config.source.tls.enabled {
-        let mut root_certs_reader =
-            BufReader::new(replicator_config.source.tls.trusted_root_certs.as_bytes());
+        let mut root_certs_reader = BufReader::new(
+            replicator_config
+                .source
+                .tls
+                .trusted_root_certs
+                .expose_secret()
+                .as_bytes(),
+        );
```

Comment:
Trusted root certs are not really a secret. They only contain the CA certificate public keys. That's why they are saved in a config map in k8s instead of a secret.

Reply:
Perfect, will remove that.